ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/Input Variables.md

```diff
@@ -1,6808 +1,6 @@
-!!! warning
-    This document describes the data model for AVD 4.x. It may or may not work in previous versions.
+---
```

Comment:
we must be missing some conditional since we are still creating this empty file.

Reply:
Wouldn't the empty line be fine? With no empty line we run into MD047 (file should end with newline character)